doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -958,7 +958,11 @@ changing the remapped destination, ``<part>`` should be used to indicate the
 component that is being modified (see `URL Parts`_), and ``<value>`` will be
 used as its replacement. You must supply a non-zero length value, otherwise
 this operator will be an effective no-op (though a warning will be emitted to
-the logs if debugging is enabled).
+the logs if debugging is enabled). Also, note that for ``PATH`` ``<part>``
+values, a leading ``/`` is not included. That is, you will generally not want
+to start your ``PATH`` value with a leading ``/``. See the ``set-destination``
```

Comment:
Perhaps ``` `Prepend a Path to the URL`_  ``` (instead of set-destination) so that we get a direct link to it? (and if there become multiple set-dest examples, this will be clearer).